src/components/SearchDialog.vue

```diff
@@ -1031,6 +1031,9 @@ export default {
 
         case 'Backspace':
         case 'Delete':
+          if (dialogService.isDialogOpened('prompt')) {
```

Comment:
That's smart 👍🏼 This issue highlight the fact that opening dialog window above another dialog window was a bad idea. But great approach to the issue ...
I would even move this right after `onKeydown(event) {` to condition the whole event handler 👌🏼